### ISSUE TITLE
Inconsistent master state

### ISSUE THREAD

**Issue by hoijui** [graphstream/gs-core#348]
It looks like [this pull request](https://github.com/graphstream/gs-core/pull/307), which had parts in _core_, _ui-swing_, _ui-javafx_ and _ui-android_ was merged into all the `dev` branches, but only ended up in `master` in _core_, and thus leaves the `master` branches of the _ui-*_ repos inconsistent with the _core_ `master` branch.

**Comment by hichbra** [comment]
Yes, we planned to fix that in the next major release, where master will be a copy of the main release, and dev will contain the updates.
